src/System.Net.WebHeaderCollection/tests/project.lock.json

```diff
@@ -3,28 +3,15 @@
   "version": -9996,
   "targets": {
     "DNXCore,Version=v5.0": {
-      "System.Collections/4.0.0-beta-22809": {
+      "System.Collections/4.0.10-beta-23024": {
         "dependencies": {
-          "System.Runtime": "4.0.0-beta-22809"
-        },
-        "compile": {
-          "ref/any/System.Collections.dll": {}
-        }
-      },
-      "System.Collections.NonGeneric/4.0.0-beta-23024": {
-        "dependencies": {
-          "System.Runtime": "4.0.20-beta-23024",
-          "System.Globalization": "4.0.10-beta-23024",
-          "System.Resources.ResourceManager": "4.0.0-beta-23024",
-          "System.Diagnostics.Debug": "4.0.10-beta-23024",
-          "System.Threading": "4.0.10-beta-23024",
-          "System.Runtime.Extensions": "4.0.10-beta-23024"
+          "System.Runtime": "4.0.20-beta-23024"
         },
         "compile": {
-          "ref/dotnet/System.Collections.NonGeneric.dll": {}
+          "ref/dotnet/System.Collections.dll": {}
```

Comment:
Strange that first build passed with this change, in all subsequent builds all tests fail with FileNotFoundException
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_prtest/2459/console
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_prtest/2450/console
I think the problem with that we depend on nongeneric IEqualityComparer which passed to NameValueCollection c-tor.
I removed this change but commit with part of other changes left with wrong commit message.
Will fix that with squashing.